recipes/jsonpatch/meta.yaml

```diff
@@ -0,0 +1,42 @@
+{% set name = "jsonpatch" %}
+{% set version = "1.16" %}
+{% set sha256 = "f025c28a08ce747429ee746bb21796c3b6417ec82288f8fe6514db7398f2af8a" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: {{ name }}-{{ version }}.tar.gz
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/{{ name }}-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+
+requirements:
+  build:
+    - python
+    - jsonpointer
+    - setuptools
+    - six
+
+  run:
+    - python
+    - jsonpointer
+    - six
+
+test:
```

Comment:
Please also test the command line scripts:

https://conda.io/docs/building/meta-yaml.html#test-commands